wire-ios-data-model/Patches/Legacy/LegacyPersistedDataPatches+Directory.swift

```diff
@@ -42,6 +46,7 @@ extension LegacyPersistedDataPatch {
         LegacyPersistedDataPatch(version: "279.0.4", block: InvalidFeatureRemoval.restoreDefaultConferenceCallingConfig),
         LegacyPersistedDataPatch(version: "285.0.0", block: ZMConversation.introduceAccessRoleV2),
         LegacyPersistedDataPatch(version: "290.0.1", block: ZMUser.refetchSelfUserDomain),
+        LegacyPersistedDataPatch(version: "297.0.2", block: UserClient.migrateAllSessionsClientIdentifiersV3)
```

Comment:
where is this version number coming from? Read from db?

Reply:
It's stored in the bundle, which is read from the `version.xcconfig` file.